### DISCUSSION TITLE
OpenDDS in bare-metal applications and softcore processors

### DISCUSSION THREAD

**Discussion by RachidFK** [OpenDDS/OpenDDS#3281] [Q&A] [answered]
Can OpenDDS functions be used in bare-metal applications? Or do they require an OS? 
Are they supported on soft microprocessors like MicroBlaze for example?

**Comment by mitza-oci** [comment, marked as answer]
> Can OpenDDS functions be used in bare-metal applications? Or do they require an OS? 

https://opendds.org/documents/building.html

> Are they supported on soft microprocessors like MicroBlaze for example?

We haven't tried it, but there's not much that's specific to a target instruction set.
